File: C++/0056. 钱币兑换.cpp
// we have defined the necessary header files here for this problem.
// If additional header files are needed in your program, please import here.

int main()
{  
  // please define the C++ input here. For example: int a,b; cin>>a>>b;;  
  // please finish the function body here.  
  // please define the C++ output here. For example:cout<<____<<endl; 
    int n;
    while(cin>>n){
        int res = 0;
        for(int i=n;i>=0;i-=3){
            res += 1;
            res += i/2;
        }
        cout<<res<<endl;
    }
    return 0;
}

File: Python/0056. 钱币兑换.py
# If you need to import additional packages or classes, please import here.

def func():
    # please define the python3 input here. 
    # For example: a,b = map(int, input().strip().split())
    # please finish the function body here.
    # please define the python3 output here. For example: print().
    while True:
        try:
            n = int(input())
            res = 0
            
            for i in range(n,-2,-3):
                res += i//2
                res += 1
            print(res)
            
        except EOFError:
            break
    
if __name__ == "__main__":
    func()
